cypress/integration/banner.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { banner } from '../support/config'
+
+describe('Banner', () => {
+  beforeEach(() => {
+    cy.visit('/')
+    expect(localStorage.getItem('bannerTimeStamp')).to.be.null
+  })
+
+  it('text correct', () => {
+    cy.get('.alert-primary.show').contains(banner.text)
+  })
+
+  it('text dismissable', () => {
+    cy.get('.alert-primary.show').contains(banner.text)
+    cy.get('.alert-primary.show').find('.fa-times').click().then(() => {
```

Comment:
Please use multiple lines, if you have builder-pattern-like chains

Reply:
fixed that